compositor_pipeline/src/pipeline/output/mp4.rs

```diff
@@ -1,8 +1,8 @@
-use std::{path::PathBuf, ptr};
+use std::{fs, path::PathBuf, ptr};
 
 use compositor_render::{event_handler::emit_event, OutputId};
 use crossbeam_channel::Receiver;
-use ffmpeg_next as ffmpeg;
+use ffmpeg_next::{self as ffmpeg};
```

Comment:
?

Reply:
I imported and removed some stuff from `ffmpeg_next` accidentally and missed that. Reverting.